orchagent/dash/dashorch.h

```diff
@@ -16,28 +18,15 @@
 #include "zmqorch.h"
 #include "zmqserver.h"
 
-struct ApplianceEntry
-{
-    swss::IpAddress sip;
-    uint32_t vm_vni;
-};
-
-struct RoutingTypeEntry
-{
-    std::string action_name;
-    std::string action_type;
-    std::string encap_type;
-    uint32_t vni;
-};
+#include "dash_api/appliance.pb.h"
+#include "dash_api/route_type.pb.h"
+#include "dash_api/eni.pb.h"
+#include "dash_api/qos.pb.h"
 
 struct EniEntry
 {
     sai_object_id_t eni_id;
-    std::string mac_address;
-    std::string qos_name;
-    swss::IpAddress underlay_ip;
-    bool admin_state;
-    std::string vnet;
+    dash::eni::Eni metadata;
```

Comment:
Can we add eni_id also to the metadata and get rid of this struct EniEntry altogether like you did with RoutingTypeEntry and ApplianceEntry?

Reply:
Oh, I don't see an opportunity to do that. Because the metadata is the protobuf message which is used by the Client. The client isn't aware of the eni_id which is the internal concept of SONiC.

Reply:
To RoutingTypeEntry and ApplianceEntry, they don't have a sai object id for identifying them, it was directly identified by the key which is aware by the Client.